importer/Monument.py

```diff
@@ -108,6 +116,12 @@ def add_statement(self, prop_name, value, quals={}, refs=[]):
             for k in quals:
                 prop_name = self.props[k]
                 qualifiers[prop_name] = quals[k]
+        if refs is None or refs is True:
+            refs = self.wlm_source
+        elif refs is False:
+            refs = None
+        if refs and not isinstance(refs, list):
```

Comment:
I would add a linebreak to separate this from the "pre-processing" if-else block